src/lib/annotations/README.md

```diff
@@ -0,0 +1,486 @@
+```
+// Turn on/off annotations for all viewers
+preview.show(..., {
+    showAnnotations: Boolean
+});
+```
+Combined with the following:
+```
+// [LEGACY] BOOLEAN - turn on/off annotations for a viewer
+preview.show(..., {
+    viewers: {
+        VIEWER_NAME: {
+            annotations: Boolean
+        }
+    }
+});
+
+// [NEW] OBJECT - turn on/off annotations for a viewer.
+preview.show(..., {
+    viewers: {
+        VIEWER_NAME: {
+            annotations: {
+                enabled: Boolean, // Enables/disables if set. Respects "showAnnotations" if empty
+                disabledTypes: String[] // List of annotation types to disable
+            }
+        }
+    }
+});
+```
+
+### Example
+Enable all annotations, turn off for Image Viewers, and disable point annotations on Document viewer:
+```
+preview.show(fileId, token, {
+    showAnnotations: true,
+    viewers: {
+        Image: {
+            annotations: {
+                enabled: false
+            }
+        },
+        Document: {
+            annotations: {
+                enabled: true,
+                disabledTypes: ['point']
+            }
+        }
+    }
+});
+```
+
+Annotators
+-------
+The name of an annotator can be one of the following `DocAnnotator` or `ImageAnnotator`. Call `boxAnnotations.getAnnotators()` to get the list of possible annotators.
+
+Additional Methods
+------------------
+`annotator.init()` initializes the annotator.
+
+`annotator.isModeAnnotatable(/* String */ type)` returns whether or not the current annotation mode is enabled for the current viewer/annotator.
+
+`annotator.showModeAnnotateButton(/* String */ currentMode)` shows the annotate button for the specified annotation mode.
+
+`annotator.getAnnotateButton(/* String */ annotatorSelector)` gets the annotation button element.
+
+`annotator.showAnnotations()` fetches and shows saved annotations.
+
+`annotator.hideAnnotations()` hides annotations.
+
+`annotator.hideAnnotationsOnPage(/* number */ pageNum)` hides annotations on a specified page.
+
+`annotator.setScale()` sets the zoom scale.
+
+`annotator.toggleAnnotationHandler()` toggles annotation modes on and off. When an annotation mode is on, annotation threads will be created at that location.
+
+`annotator.disableAnnotationMode(/* String */ mode, /* HTMLElement */ buttonEl)` disables the specified annotation mode.
+
+`annotator.enableAnnotationMode(/* String */ mode, /* HTMLElement */ buttonEl)` enables the specified annotation mode.
+
+Events
+------
+The annotator object exposes `addListener` and `removeListener` for binding to events. Event listeners should be bound before `showAnnotations()` is called, otherwise events can be missed.
```

Comment:
I would reword this to "Events can be bound to with `addListener` and removed with `removeListener`."